### PR TITLE
Shellcheck GitHub actions workflow 1

### PR DESCRIPTION
**Pull request by fcaffieri**
|Related issue|
|---|
|#1351|

## Description
Generating the artifacts according to the changes obtained in the PR on the unattended_installer directory.
Generate the scripts with the builder.sh to later analyze them with ShellCheck

## Tests

[Test Workflow poc](https://github.com/wazuh/wazuh-packages/pull/1366)
[Test Workflow this PR](https://github.com/wazuh/wazuh-packages/pull/1385/checks)

### CODE REVIEW

**Review by rauldpm — APPROVED** [review]
LGTM

File: .github/workflows/pr-reviews.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: pr-review
```
Comment (alberpilot) [inline .github/workflows/pr-reviews.yml:1]: Shellcheck code analysis

File: .github/workflows/pr-reviews.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.1.0
+      - name: Set Artifacts
+        uses: jitterbit/get-changed-files@v1
+        id: changed_files
+        with:
+          format: 'csv'
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (alberpilot) [inline .github/workflows/pr-reviews.yml:15]: Test if necessary. `secrets.GITHUB_TOKEN`

File: .github/workflows/pr-reviews.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      matrix: 
+        tool: ${{ fromJson(needs.get-artifacts.outputs.tools_matrix) }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.1.0
+      - name: Generate artifacts
+        uses: './.github/actions/generate-artifacts/'
+        with: 
+          artifact: "${{ matrix.tool }}"
+      - name: Checkout
```
Comment (alberpilot) [inline .github/workflows/pr-reviews.yml:40]: Remove it

**Review by alberpilot — CHANGES_REQUESTED** [review]

**Pull request closed without merging**